### DISCUSSION TITLE
Support padStart

### DISCUSSION THREAD

**Discussion by anonsyn** [barvian/number-flow#23] [Help] [answered]
Currently, the library uses `Intl.NumberFormat`, but it doesn't support padding numbers with leading zeros. Adding support for padding numbers (e.g., using `padStart`) would help maintain consistent digit length during animations.

**Example:**
```js
// Desired output:
001, 002, 010, etc.
```
This would improve visual consistency in number animations. Thank you!

**Comment by barvian** [comment, marked as answer]
Hi! I think the [`minimumIntegerDigits` option](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#minimumintegerdigits) should help with this: https://codesandbox.io/p/sandbox/numberflow-padded-frqdxv

**Comment by anonsyn** [comment]
Thank you for the quick support. You’re my lifesaver
